build/isc-bind9/local.mog

```diff
@@ -0,0 +1,59 @@
+#
+# The contents of this file are subject to the terms of the
+# Common Development and Distribution License, Version 1.0 only
+# (the "License").  You may not use this file except in compliance
+# with the License.
+#
+# You can obtain a copy of the license at usr/src/OPENSOLARIS.LICENSE
+# or http://www.opensolaris.org/os/licensing.
+# See the License for the specific language governing permissions
+# and limitations under the License.
+#
+# When distributing Covered Code, include this CDDL HEADER in each
+# file and include the License file at usr/src/OPENSOLARIS.LICENSE.
+# If applicable, add the following below this CDDL HEADER, with the
+# fields enclosed by brackets "[]" replaced with your own identifying
+# information: Portions Copyright [yyyy] [name of copyright owner]
+
+# Copyright 2011-2012 OmniTI Computer Consulting, Inc.  All rights reserved.
+# Copyright (c) 2013 by Delphix. All rights reserved.
+# Copyright 2020 OmniOS Community Edition (OmniOSce) Association.
+
+# Mediated binaries
+<transform file path=$(PREFIX)/(s?bin)/(.*) -> emit \
+    link path=$(OPREFIX)/%<1>/%<2> target=../$(PROG)-$(VERSION)/%<1>/%<2> \
+    mediator=$(PROG) mediator-version=$(VERSION) >
+
+# Mediated man pages
+<transform file path=$(PREFIX)/share/man/(man\d)/(.*) -> emit \
+    link path=$(OPREFIX)/share/man/%<1>/%<2> \
+    target=../../../$(PROG)-$(VERSION)/share/man/%<1>/%<2> \
+    mediator=$(PROG) mediator-version=$(VERSION) >
+
+<transform file path=$(PREFIX)/sbin/ \
+    -> set restart_fmri svc:/network/dns/$(PROG)$(sVERSION):default>
+
+dir group=bin mode=0755 owner=root path=etc/$(PREFIX)
+<transform file path=etc/$(PREFIX)/ -> set preserve renamenew>
+
+dir group=named mode=0755 owner=named path=var/$(PREFIX)/namedb
+dir group=named mode=0755 owner=named path=var/$(PREFIX)/namedb/master
+
+# Additional files
+file ./tmp/named-911.conf \
```

Comment:
Can you use `$(sVERSION)` here instead of `911`?

Reply:
this is also done, and searched and fixed other places this can be changed.